package NYT.NTableClient.NProto;

import "yt_proto/yt/client/table_chunk_format/proto/chunk_meta.proto";

////////////////////////////////////////////////////////////////////////////////

message TRlsReadSpec
{
    message TTrivialDeny {}

    oneof expression_or_trivial_deny {
        string expression = 1;
        TTrivialDeny trivial_deny = 2;
    }

    optional TTableSchemaExt table_schema = 3;
}

////////////////////////////////////////////////////////////////////////////////
